examples/tutorial.ipynb

```diff
@@ -125,15 +118,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `report()` method helps you quickly understand the major issues detected in the dataset. It reports the number of images in the dataset that exhibit each type of issue, and shows example images corresponding to most the severe instances of each issue."
+    "The `imagelab.report()` method helps you quickly understand the major issues detected in the dataset. It reports the number of images in the dataset that exhibit each type of issue, and shows example images corresponding to the most severe instances of each issue."
```

Comment:
```suggestion
    "The `report()` method helps you quickly understand the major issues detected in the dataset. It reports the number of images in the dataset that exhibit each type of issue, and shows example images corresponding to the most severe instances of each issue."
```

Reply:
you do not need to include name of variable when discussing an instance method, when it is obvious from the code cell